src/components/containers/Wallet/TransactionDetails.css

```diff
@@ -25,6 +25,8 @@
 .transactionDeyItemContent {
```

Comment:
Do you know what is `dey`?
I would ask you to fix it to `day`, but checking the code where is it used as className it does not seem to be `datetime` related data.